include/deal.II/fe/fe_base.h

```diff
@@ -242,6 +242,15 @@ class FiniteElementData
    */
   const unsigned int dofs_per_hex;
 
+  /**
+   * Number of non-zero degrees of freedom (DoFs) on the cell that,
+   * however, can not be associated with the specific geometric object (such as
+   * face or edge). In these cases, DoFs are typically globally distributed over
+   * (patches of) the Triangulation. Typical examples are IGA, XFEM, Catmull
+   * Clarks FE etc.
```

Comment:
Catmull Clarks -> Catmull-Clark's

Reply:
I think that it would be most useful to have each of these abbreviations spelled out in full.

My overall preference would be to have these items in a bulleted list with a reference to some relevant Wikipedia article, or a description of the algorithm, or something else. There is, for example, a wiki article on [Catmull-Clark subdivision surfaces](https://en.wikipedia.org/wiki/Catmull%E2%80%93Clark_subdivision_surface), but is this the "FE" that is referred to here? Some more details would be enlightening.

Reply:
I have made some changes to the documentation and refer IGA, XFEM and Catmull-Clark's subdivision surfaces to three well-known papers.But  I am not sure it is the right way to do this. Can you advise me how to add those references in deal.ii? I am sorry that I am new in code developing.

Reply:
@zhaoweiliu : you can do that by writing
```
<a href="https://...">IGA</a>
```
and alike in documentation.